Batch.py

```diff
@@ -58,28 +61,28 @@ def data_into_batch(self, data):
         camera_data = camera_data.cuda().float() / 255. - 0.5
```

Comment:
The torch.cat in line 60 is one of the cats that is taking the most of the time, I think it would be better to insert this into the prebuilt tensor in the loop.